[2022-04-13 22:35:58,510] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:20+00:00 [queued]>
[2022-04-13 22:35:58,531] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:20+00:00 [queued]>
[2022-04-13 22:35:58,532] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:35:58,533] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:35:58,534] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:35:58,563] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:20+00:00
[2022-04-13 22:35:58,569] {standard_task_runner.py:52} INFO - Started process 11225 to run task
[2022-04-13 22:35:58,579] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:20+00:00', '--job-id', '1933', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp2zixfthn', '--error-file', '/tmp/tmpms0tcves']
[2022-04-13 22:35:58,582] {standard_task_runner.py:77} INFO - Job 1933: Subtask ingest
[2022-04-13 22:35:58,702] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:20+00:00 [running]> on host 13db72185839
[2022-04-13 22:35:58,797] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:20+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:20+00:00
[2022-04-13 22:35:58,799] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:35:58,800] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:35:58,811] {subprocess.py:85} INFO - Output:
[2022-04-13 22:35:58,816] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:35:58,817] {subprocess.py:89} INFO - dags
[2022-04-13 22:35:58,817] {subprocess.py:89} INFO - logs
[2022-04-13 22:35:58,818] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:35:58,818] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:35:58,818] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:35:58,819] {subprocess.py:89} INFO - plugins
[2022-04-13 22:35:58,819] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:35:58,819] {subprocess.py:89} INFO - scripts
[2022-04-13 22:35:58,820] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:35:58,820] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:35:58,865] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060020, start_date=20220413T223558, end_date=20220413T223558
[2022-04-13 22:35:58,909] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:35:58,966] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
